src/test/java/com/team4/testingsystem/services/TestsServiceTest.java

```diff
@@ -0,0 +1,158 @@
+package com.team4.testingsystem.services;
+
+import com.team4.testingsystem.entities.TestEntity;
+import com.team4.testingsystem.entities.User;
+import com.team4.testingsystem.repositories.TestsRepository;
+import com.team4.testingsystem.repositories.UsersRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+
+@ExtendWith(MockitoExtension.class)
+class TestsServiceTest {
+
+private final String DATE_FOR_TESTS = "2020-01-01T10:00:00";
+
+    @Mock
+    User user;
+
+    @Mock
+    TestEntity testEntity;
+
+    @Mock
+    UsersRepository usersRepository;
+
+    @Mock
+    TestsRepository testsRepository;
+
+    @InjectMocks
+    TestsService testsService;
+
+    @Test
+    void successfulGetting() {
+
+        Mockito.when(testsRepository.existsById(1L)).thenReturn(true);
+        Assertions.assertNotNull(testsService.getTestById(1));
+
```

Comment:
Pretty strange test. According to its name, it should verify that you receive an object from the repo. 
Please, also check the other tests with the same reason

Reply:
The test should call a method, receive the response and validate, if it is the sane data we expected